### ISSUE TITLE
NULL attributes should be excluded

### ISSUE THREAD

**Issue by razvanphp** [yiisoft/yii2-sphinx#29]
It seems like Sphinx doesn't like NULL values, nor does know anything about those parameters.

Source: http://sphinxsearch.com/forum/view.html?id=8052

The error I receive:
````sql
SQLSTATE[42000]: Syntax error or access violation: 1064 sphinxql: syntax error, unexpected NULL, expecting CONST_INT (or 5 other tokens) near 'NULL)'
The SQL being executed was: REPLACE INTO `rt` (`id`, `user_id`, `first_name`, `last_name`, `phone`) VALUES (1798, '1798', 'Dasdasd', 'Fadas', NULL)

Error Info: Array
(
    [0] => 42000
    [1] => 1064
    [2] => sphinxql: syntax error, unexpected NULL, expecting CONST_INT (or 5 other tokens) near 'NULL)'
)
````

I would propose to automatically skip those attributes when setting them, like the indexer command line does with the disk-index.

**Comment by klimov-paul** [comment]
Why do you inserting `null` value into runtime index?
I don't see reason, why `Command` should hide possible developer mistake by skipping `null` values automatically.

**Comment by razvanphp** [comment]
It's not actually a mistake, fields in the DB can be null, and the default `indexer` command of sphinx silently ignores those values. For the RT index, as you know, we start with an empty index and we need to programmatically insert the values there. 

For <i>convenience</i> it would be good that the SDK helps you and strips those fields from the query, since sphinx will always throw an error in this case, and you cannot search for NULL values and attributes anyway.

**Comment by klimov-paul** [comment]
Issue resolved by commit 741c353
